### PR TITLE
Module 3

### PR DESCRIPTION
**Pull request by mberkii**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -16,31 +80,45 @@ const genres = [
 ]
 
 function App() {
-    const [selected, setSelected] = useState(genres[0].name);
+    const [selectedGenre, setSelectedGenre] = useState(genres[0].name);
+	const [selectedMovie, setSelectedMovie] = useState();
```
Comment (EpamFanAnton) [inline src/App.js:84]: looks like something happened with tabulation here

File: src/App.js [hunk 2]
```diff
@@ -16,31 +80,45 @@ const genres = [
...
+						genres={genres}
+						selected={selectedGenre}
+						onClick={onSelect}
+					/>
+					<SortControl setSortedBy={setSortedBy} />
+				</div>
+				<p className="text-left mb-2"><b>{movies?.length}</b> movies found</p>
+				<div className="d-flex">
+					{moviesList.map((movie) =>
+						<MovieTile key={movie.id} details={movie} selectMovie={setSelectedMovie}/>
```
Comment (EpamFanAnton) [inline src/App.js]: can't see onClick handler mentioned in the task description "Additionally, the component should receive a callback property to capture click event."

File: src/App.js [hunk 3]
```diff
@@ -16,31 +80,45 @@ const genres = [
...
 function App() {
-    const [selected, setSelected] = useState(genres[0].name);
+    const [selectedGenre, setSelectedGenre] = useState(genres[0].name);
+	const [selectedMovie, setSelectedMovie] = useState();
+	const [sortedBy, setSortedBy] = useState();
 
-	const onSelect = (event) => {
-		event.preventDefault()
-		setSelected(event.target.dataset.name)
+	const sortMovies = () => {
```
Comment (EpamFanAnton) [inline src/App.js]: this name is not descriptive. this function returns sorted movies, not sorting them.

**Review by EpamFanAnton — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (mberkii) [reply src/App.js:84]: In VS Code everything is good, probably github misinterpreted these lines
image

**Review by EpamFanAnton — APPROVED** [review]

**Pull request merged**